### ISSUE TITLE
Unable to edit alt text

### ISSUE THREAD

**Issue by nibbles-** [pachli/pachli-android#1380]
**Describe the bug**
Toot a toot with an image in it.
Edit the toot and edit the alt text of the image.
When reposting you will get an error message stating that the feature is disabled or that is was not possible to connect to the server.
It varies.
Tested on mastodon.social and it is possible to edit the alt text from the web gui.

**Expected behavior**
The alt text should be edited and the post should be updated.

**Versions**
Pachli 2.10.1

Device:

Google Pixel 7
Android version: 15
SDK level: 35

Account:
mastodon.social
Version: 4.4.0-nightly.2025-03-14-security

**Affirmation**
I have checked other issues for this project (open and closed) and I cannot find one that matches the problem I am reporting.
